README.md

```diff
@@ -1 +1,82 @@
-# restaurante-page-v1
\ No newline at end of file
+<h3 align="center">Restaurant - Javascript</h3>
+
+<div align="center">
+
+[![Status](https://img.shields.io/badge/status-active-success.svg)](https://github.com/marcelomaidden/restaurant-page-v1)
+[![Github Issues](https://img.shields.io/badge/GitHub-Issues-orange)](https://github.com/marcelomaidden/restaurant-page-v1/issues)
+[![GitHub2 Pull Requests](https://img.shields.io/badge/GitHub-Pull%20Requests-blue)](https://github.com/marcelomaidden/restaurant-page-v1/pulls)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](/LICENSE)
+</div>
+<p align="center">Restaurant in JavaScript is a practice project, which includes managing the dynamically DOM manipulation. I used Javascript, HTML5, CSS3, Bootstrap, Font Awesome to implement it</p>
+
+
+## Live Link Demo
+
+[Live Link Demo](https://marcelomaidden.github.io/restaurant-page-v1/)
+
+## Built With
+
+- HTML5
+- CSS3
+- Bootstrap
+- JavaScript
+- Font Awesome
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Text Editor (VSCode is suggested.)
+
+
+### Setup
+
+- Install [git](https://git-scm.com/downloads)
+- Open Terminal
+- Change directory to folder to download repository
+- Use `cd <file-path>`
+- Run `git clone https://github.com/marcelomaidden/restaurant-page-v1`
```

Comment:
### [Optional]
I trust you can do this before merging.

I think you can add one more step to this setup;

It would be helpful to add the following;
1. open `index.html` in the dist folder.

_newbie would be happy with this 😄_